src/main/resources/assets/js/models/Collection.js

```diff
@@ -0,0 +1,27 @@
+var Backbone = require("backbone");
+
+module.exports = Backbone.Collection.extend({
```

Comment:
I've tried it here:
https://github.com/mesosphere/marathon/compare/325-make-resources-linkable_cleaned...325-make-resources-linkable_updatePolling
Don't makes sense for me for this case I handle with the fetching status, because I need this line to fetch the tasks immediately:
https://github.com/mesosphere/marathon/blob/325-make-resources-linkable_cleaned/src/main/resources/assets/js/components/TaskListComponent.jsx#L30
Trigger with:
http://localhost:8080/#apps/:yourappid (reload the page to get a fresh coming in status)

If I trigger indirectly by clicking on the app in the applist I am getting multiple http-requests of the same resource, that's bad.

Don't get it how to prevent this without the fetching status..